packages/os/os.spec

```diff
@@ -97,6 +98,11 @@ Summary: Setting generator for templated settings values.
 %description -n %{_cross_os}schnauzer
 %{summary}.
 
+%package -n %{_cross_os}shibaken
+Summary: Setting generator for user-data with IMDS public keys.
```

Comment:
Or even more general: `Settings generator for populating host-container user-data vended by IMDS`

We'll expect to handle other data and the `Summary` tag isn't made visible. Making this general means we won't have to remember to update the description as features are added elsewhere in the tree (and the suggested wording is not inaccurate as-is).

Reply:
I'll change this to:
`Summary: Setting generator for populating host container user-data with IMDS.`

That way it will remain general and is also <80 chars.